libr/anal/var.c

```diff
@@ -702,7 +702,7 @@ R_API void extract_rarg(RAnal *anal, RAnalOp *op, RAnalFunction *fcn, int *reg_s
 				!strcmp (opdreg, regname);
 			if ((op->src[0] && opsreg && !strcmp (opsreg, regname)) || cond) {
 				const char *vname = NULL;
-				const char *type = "int";
+				const char *type = NULL;
```

Comment:
should be `strdup ("int")`

Reply:
As said by @radare in another commit, use `type ? type : "int"` in the line when `type` is used. This, however, should be `char *` AFAIK, given it's used as argument to free later.